flink-tests/src/test/scala/org/apache/flink/api/scala/migration/StatefulJobSavepointMigrationITCase.scala

```diff
@@ -115,7 +115,7 @@ class StatefulJobSavepointMigrationITCase(
         }
       )
       .flatMap(new StatefulFlatMapper)
-      .addSink(new AccumulatorCountingSink)
+      .addSink(new AccumulatorCountingSink[(Long, Long)])
```

Comment:
What's up with the changes in the Scala tests?

Also, we didn't add the API method on the Scala DataSteam yet, right?

Reply:
It is because that the scala compiler cannot recognize these two overloaded `addSink` methods